cmd/troubleshoot/cli/run.go

```diff
@@ -153,41 +153,29 @@ func runTroubleshoot(v *viper.Viper, arg string) error {
 		Namespace:                 v.GetString("namespace"),
 		ProgressChan:              progressChan,
 		SinceTime:                 sinceTime,
-	}
-
-	archivePath, err := supportbundle.CollectSupportBundleFromSpec(&supportBundle.Spec, additionalRedactors, createOpts)
```

Comment:
all changes made here are part of this function now https://github.com/replicatedhq/troubleshoot/pull/417/files#diff-04328572fa60f5b4cdaaa40835e6d53387d06e87f3e4becf95ab471678323bb6R41